server/tables/tables.go

```diff
@@ -531,7 +527,7 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
 	}
 
 	// Migrate Groups.APIKey to APIKey rows.
```

Comment:
offtopic: is it time to get rid of this?

Reply:
I think maybe we can consider removing DB migrations as part of each major BB release, and having a separate tool to help customers upgrade their DB from older to newer DB versions between upgrades. (e.g. to upgrade from V1 to V2 to V3, the tool would check out the last BB release for each version then run the DB migration logic from each release).